tools/lisa-build-asset

```diff
@@ -84,28 +126,32 @@ def make_asset(asset, arch_list, build_dir, recipe_dir, toolchains):
         cm = tempfile.TemporaryDirectory()
 
     with cm as build_dir:
-        # All architectures need to be built
-        if len(arch_list) > 1:
-            # Download sources once for all architectures
-            download_dir = os.path.join(build_dir, 'download')
-            os.makedirs(download_dir)
-            make(asset, ['download'], arch='any', toolchain=None, recipe_dir=recipe_dir, build_dir=download_dir)
-
-            # Build and install for each
-            actions = ['build', 'install']
-            for arch in arch_list:
-                arch_build_dir = os.path.join(build_dir, arch)
-                shutil.copytree(download_dir, arch_build_dir)
-
-                toolchain = toolchains[arch]
-                make(asset, actions, arch, toolchain, recipe_dir, arch_build_dir)
-
-        # Only one architecture was requested
-        else:
-            arch = arch_list[0]
-            actions = ['download', 'build', 'install']
+        alpine_source_dir = None
+        if native_build:
```

Comment:
to be done in install_base.sh --install-toolchain switch

Reply:
Since the alpine-chroot-install repository is only used in this script, I am not setting a LISA_... env variable for it

Reply:
Not sure what the code was but I suppose it was the build dependencies.